.flake8

```diff
@@ -1,4 +1,5 @@
 [flake8]
-exclude=.tox,utils,inventory
+# TODO: cleanup flake8 issues with utils/test/*
```

Comment:
No, didn't want to add any more scope creep. Figured it was better to handle as we refactor the tests to take advantage of pytest.